clients/mobile/src/authentication/login/hooks/useLogin.ts

```diff
@@ -63,17 +68,39 @@ export const useLogin = (): any => {
 
   const handleGoogleSignIn = async(): Promise<void> => {
     try {
-      await googleLogin()
-      navigation.navigate(SCREENS.PROFILE)
+      const isGoogleSignedIn = await googleLogin()
+      if (isGoogleSignedIn) {
+        auth?.setIsAuthenticated(true)
```

Comment:
same

Reply:
Remove the chaining operator. [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/239/commits/4356c490c82c28f3876f0b0d21519b1caaff1826)

```
const { setIsAuthenticated } = useAuth()
setIsAuthenticated(true)
```